packages/application-components/src/components/modal-pages/form-modal-page/README.md

```diff
@@ -50,6 +50,7 @@ Form Modal pages are controlled components used to render a page with a form or
 | `dataAttributesPrimaryButton`   | `object`                   |    -     | -                        | -                                                                                                                                                                                                                                                                                  | Use this prop to pass `data-` attributes to the primary button |
 | `getParentSelector`             | `function`                 |    -     | -                        | -                                                                                                                                                                                                                                                                                  | The function should return an HTML element that will be used as the parent container to hold the modal DOM tree. If no function is provided, it's expected that an HTML element with the `id="parent-container"` is present in the DOM. In `NODE_ENV=test` environment, the default HTML element is `body`. |
 | `customControls`                | `node`                     |    -     | -                        | Pass a React.node to be used in place of the pre-determined controls. This can be useful if you need actions other than Cancel & Confirm, or other types of buttons, while keeping the same modal header layout                                                                    |
+| `shouldDelayOnClose`            | `bool`                     |    -     | `true`                   | Sets wether the ModalPage should delay calling its onClose function to allow the closing animation to finish. This can be turned off if the developer is controlling the ModalPage only through the `isOpen` prop, and not abruptly mounting/unmounting it or one of its parent elements. |
```

Comment:
```suggestion
| `shouldDelayOnClose`            | `bool`                     |    -     | `true`                   | Sets weather the ModalPage should delay calling its onClose function to allow the closing animation time to finish. This can be turned off if the developer is controlling the ModalPage only through the `isOpen` prop, and not abruptly mounting/unmounting it or one of its parent elements. |
```

Reply:
🤨
![image](https://user-images.githubusercontent.com/2941328/59682317-45b48c80-91d6-11e9-80b3-0fa6f1734391.png)
TIL 😄